scripts/src/main/resources/scripts/command/ide

```diff
@@ -320,30 +320,35 @@ function doUpdateProjects() {
     last_cwd="${PWD}"
     for projectconfig in "${SETTINGS_PATH}"/projects/*.properties; do
       doEcho "Importing from ${projectconfig}..."
-      project_path=$(grep "^path" "$projectconfig" | cut -d'=' -f2)
-      project_workingsets=$(grep "^workingsets" "$projectconfig" | cut -d'=' -f2)
-      project_workspace=$(grep "^workspace" "$projectconfig" | cut -d'=' -f2)
-      project_git_url=$(grep "^git.url" "$projectconfig" | cut -d'=' -f2)
-      project_git_branch=$(grep "^git.branch" "$projectconfig" | cut -d'=' -f2)
-      project_build_path=$(grep "^build.path" "$projectconfig" | cut -d'=' -f2)
-      project_eclipse=$(grep "^eclipse" "$projectconfig" | cut -d'=' -f2)
+      project_path=$(grep "^path" "$projectconfig" | cut -d'=' -f2-)
+      project_workingsets=$(grep "^workingsets" "$projectconfig" | cut -d'=' -f2-)
+      project_workspace=$(grep "^workspace" "$projectconfig" | cut -d'=' -f2-)
+      project_git_url=$(grep "^git.url" "$projectconfig" | cut -d'=' -f2-)
+      project_git_branch=$(grep "^git.branch" "$projectconfig" | cut -d'=' -f2-)
+      project_build_path=$(grep "^build.path" "$projectconfig" | cut -d'=' -f2-)
+      project_eclipse=$(grep "^eclipse" "$projectconfig" | cut -d'=' -f2-)
+      project_build_params=$(grep "^build.params" "$projectconfig" | cut -d'=' -f2-)
     
       [ -z "${project_path}" ] && doFail "Config is missing required parameter 'project.path'"
       [ -z "${project_workspace}" ] && project_workspace=main
       [ -z "${project_git_url}" ] && doFail "Config is missing required parameter 'project.git.url'"
       [ -z "${project_eclipse}" ] && project_eclipse=import
+      [ -z "${project_build_path}" ] && project_build_path=.
     
       doDebug "Project (Path: ${project_path}, Url: ${project_git_url}, Branch: ${project_git_branch}, Eclipse: ${project_eclipse}, Workspace: ${project_workspace}, Workingsets: ${project_workingsets})"
       doDebug "Pull or clone git project ${project_path}..."
       project_workspace_path="${DEVON_IDE_HOME}/workspaces/${project_workspace}"
       mkdir -p "${project_workspace_path}"
       cd "${project_workspace_path}" || exit 255
       doGitPullOrClone "${project_path}" "${project_git_url}" "${project_git_branch}"
-      if [ -n "${project_build_path}" ]
+      doDebug "Building project with params: ${project_build_params}"
+      if [ -n "${project_build_params}" ]
       then
         cd "${project_path}/${project_build_path}" || exit 255
-        doDevonCommand build
+        doDevonCommand build ${project_build_params}
```

Comment:
But that's just passing parameters.
As of our last discussion, I even thought of running doDevonCommand ${project_post_build_command} and let the user specify what to do. So a user could even decide to do something like `py doSomeGreatSetup`.

Even the question is here whether we should also support anything like also git bash standard commands, such that I could process something like an sed command or anything like this.

Reply:
The problem is, that a user could also provide `rm -rf /` as build command then...

Reply:
Ok. My view on that is simple:
With great power comes great responsibility. I don't think that we have to discuss about attackers but just about faulty configurations. But in my opinion it's quite obvious that you should not put here any cd / && rm -Rf :)
But for me it would be even ok if we simply skip execution possibilities of any non devonfw command.

Reply:
Then you would propose:
`doDevonCommand ${project_build_cmd}`

Reply:
yes

Reply:
et voilà!